src/modules/folder/folder.controller.ts

```diff
@@ -525,4 +526,21 @@ export class FolderController {
 
     return { size };
   }
+
+  @Put('move/:folderUuid/to/:destinationUuid')
```

Comment:
Same here, this is a partial update where you are updating a folder's attribute by uuid. It's more semantic to just use PATCH /:uuid, the new parentUuid should be sent on the body, and if you can use the field name as is, the better. If in the future we modify more parameters, we must be handling the usecase routing here.